environments/india/inventory.ini

```diff
@@ -143,8 +145,9 @@ kafka_b3
 
 [celery:children]
 celery7
-celery8
-celery9
+celerybeat_a1
```

Comment:
@pvisweswar You can remove `celerybeat_a1` from here. We will add it later after we have configure `celery12` and `celery13` else you will keep seeing the test failures.
We run deploy stack on this machine only when it is added to celery group later.